### PR TITLE
[SPARK-6643][MLLIB] Implement StandardScalerModel missing methods

### PR DESCRIPTION
**Pull request by Lewuathe**
This is the sub-task of SPARK-6254.
Wrap missing method for `StandardScalerModel`.

### CODE REVIEW

File: mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala [hunk 1]
```diff
@@ -434,8 +434,39 @@ private[python] class PythonMLLibAPI extends Serializable {
...
+     * @return
+     */
+    def setWithMean(withMean: Boolean): this.type = {
+      model.setWithMean(withMean)
+      this
+    }
+
+    /**
+     * Setter of the boolean which decides
+     * whether it uses mean or not
```
Comment (viirya) [inline mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala]: uses mean or not -> uses std or not?

File: python/pyspark/mllib/feature.py [hunk 2]
```diff
@@ -132,6 +132,22 @@ def transform(self, vector):
...
+        Setter of the boolean which decides
+        whether it uses mean or not
+        """
+        self.call("setWithMean", withMean)
+        return self
+
+    def setWithStd(self, withStd):
+        """
+        Setter of the boolean which decides
+        whether it uses mean or not
```
Comment (viirya) [inline python/pyspark/mllib/feature.py]: same here. mean -> std.

**Comment by Lewuathe** [comment]
@viirya Thank you for comment. I updated.

File: mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala [hunk 3]
```diff
@@ -434,8 +434,39 @@ private[python] class PythonMLLibAPI extends Serializable {
     new Normalizer(p).transform(rdd)
   }
 
+  private[python] class StandardScalerModelWrapper(model: StandardScalerModel)
```
Comment (mengxr) [inline mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala]: @Lewuathe The wrapper doesn't do anything different that `StandardScalerModel`. Do we really need it?

**Comment by Lewuathe** [comment]
@mengxr Yes. If I think again, it is unnecessary. Thank you for pointing out. I updated.

**Comment by mengxr** [comment]
LGTM. Let's retry Jenkins.

test this please

**Comment by mengxr** [comment]
test this please

**Comment by mengxr** [comment]
@Lewuathe You need an extra empty line to pass python style checker. Try `dev/lint-python` on your local and see the error message.

**Comment by Lewuathe** [comment]
@mengxr I updated. Thank you!

**Comment by mengxr** [comment]
Merged into master. Thanks!

**Pull request closed without merging**